### DISCUSSION TITLE
Global value mappings

### DISCUSSION THREAD

**Discussion by PhilippDehner** [grafana/grafana#39126] [Dashboards, Panels & Visualizations]
I'd like to do the same value mapping in many panels and dashboards. So it would be nice, if they could be defined one time. So they could be used multiple times. 
 
What do you mean?

It could be done by #39127, but that would not be the best solution, because in this case I also have to create every mapping, only the values are automatically given

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
